### PR TITLE
Cb 3330 data import

### PR DESCRIPTION
**Pull request by DenisSinelnikov**

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/stream/importer/DataImporterCSV.java [hunk 1]
```diff
@@ -245,7 +245,7 @@ public void runImport(@NotNull DBRProgressMonitor monitor, @NotNull DBPDataSourc
                             break;
                         }
 
-                        if (line.length < targetAttrSize) {
+                        if (line.length <= targetAttrSize) {
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/stream/importer/DataImporterCSV.java]: Why? It doesn't make sense when `line.length == targetAttrSize`

File: plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/database/DatabaseTransferConsumer.java [hunk 2]
```diff
@@ -1004,4 +1004,12 @@ public DBPDataKind getDataKind() {
...
         }
     }
+
+    public void setSettings(DatabaseConsumerSettings settings) {
+        this.settings = settings;
+    }
+
+    public void setContainerMapping(DatabaseMappingContainer containerMapping) {
+        this.containerMapping = containerMapping;
+    }
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/database/DatabaseTransferConsumer.java]: Please specify `@NotNull` / `@Nullable` annotations.

File: plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/stream/importer/DataImporterCSV.java (hunk 1 above)
Comment (DenisSinelnikov) [reply plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/stream/importer/DataImporterCSV.java]: Added for testing, forget to remove.

**Review by ShadelessFox — APPROVED** [review]

File: plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/database/DatabaseTransferConsumer.java [hunk 3]
```diff
@@ -1004,4 +1004,12 @@ public DBPDataKind getDataKind() {
             return this.binding.getDataKind();
         }
     }
+
+    public void setSettings(@Nullable DatabaseConsumerSettings settings) {
```
Comment (alexander-skoblikov) [inline plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/database/DatabaseTransferConsumer.java:1008]: why isn't 'initTransfer' being used?

Comment (alexander-skoblikov) [inline plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/database/DatabaseTransferConsumer.java:1014]: why isn't 'initTransfer' being used?

**Review by alexander-skoblikov — CHANGES_REQUESTED** [review]

**Review by alexander-skoblikov — APPROVED** [review]

**Pull request merged**